hazelcast/src/test/java/com/hazelcast/client/test/ClientTestSupport.java

```diff
@@ -82,6 +85,18 @@ protected HazelcastClientInstanceImpl getHazelcastClientInstanceImpl(HazelcastIn
         return clientProxy.client;
     }
 
+    protected void makeSureDisconnectedFromServer(final HazelcastInstance client, UUID memberUUID) {
+        assertTrueEventually(() -> {
+            ClientConnectionManager connectionManager = getHazelcastClientInstanceImpl(client).getConnectionManager();
```

Comment:
We can use the `getConnection(UUID uuid)` here instead of iterating over the active connections. Also, there is a possible leftover `System.out.println` below